tests/yaml/comment/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -336,6 +437,107 @@ printWidth: 80
 ================================================================================
 `;
 
+exports[`sequence.yml format 1`] = `
+====================================options=====================================
+parsers: ["yaml"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+-  - a
+
+   # - b
+
+   # - c
+
+   - e
+
+-  - a
+
+   # - b
+
+   # - c
+
+=====================================output=====================================
+- - a
+
+  # - b
+
+  # - c
+
+  - e
+
+- - a
+
+  # - b
+
+  # - c
+
+================================================================================
+`;
+
+exports[`sequence-2.yml format 1`] = `
+====================================options=====================================
+parsers: ["yaml"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+- before
+
+  # before.comment
+- after
+  # after.comment
+
+- before-after
+
+  # before-after.comment
+
+- none:
+  # none.comment
+- before(2 line)
+
+
+  # before.comment
+- after(2 line)
+  # after.comment
+
+
+- before-after(2 line)
+
+
+  # before-after.comment
+
+
+- none(2)
+  # none.comment
+=====================================output=====================================
+- before
+
+# before.comment
```

Comment:
This is strange!!!

**Prettier pr-9143**
[Playground link](https://deploy-preview-9143--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEAzCEkB0oAJcDEukAtiQjDgEYCGATtnocRGRTjiADQgQAOMAJbQAzslD06EAO4AFegjEoaAG2k0AnmJ5U6NMAGs4MAMp99gqAHNkMOgFc4POCSpwAJu48AZGtfs0VnAAYhB0JDQwQtbIIDT2MBDcIAAWMCQqAOopgvAi5mBwJoq5ggBuuRqxYCLaIJYicHQwsnpWEcioqo08AFYiAB4AQnqGxiY05N6WcJ3dTiD9AyaWVipwAIr2EPBzKj0g5nSNdLEakyrJfHSWMJmC7jApyAAcAAw81xCNmXp8sdc4CcyrMeABHbbwVr8JRxEQAWigcA8HmSdDgEME6NagQ6SC6+wWjRIglsDiJq3WWx2s3x8x4MBoVHuj2eSAATAy9IIVKsAMKsPEgIEAVmS9kaABUmUoCQcyo4AJJQLywExgG4CACCKpMMA06z2jQAvsagA)
```sh
--parser yaml
```

**Input:**
```yaml
foo:
  # comment
bar:
  # comment


```

**Output:**
```yaml
foo:
  # comment
bar:
  # comment

```

**Prettier pr-9143**
[Playground link](https://deploy-preview-9143--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEAzCEkB0oAJcDEukAtiQjDgEYCGATndnocRGRTjiADQgQAOMAJbQAzslD06EAO4AFegnEoaAGxk0AnuN5U6NMAGs4MAMr8DQqAHNkMOgFc4vOCSpwAJh88AZGjYcaazgAMQg6EhoYYRtkEBoHGAgeEAALGBJVAHVUoXhRCzA4UyU8oQA3PM04sFEdECtRODoYOX1rSORUNSbeACtRAA8AIX0jE1Mach8rOC6e5xABwdMra1U4AEUHCHh51V6QCzomujjNKdUU-jorGCyhDxhU5AAOAAZeG4gmrP1+OI3OCncpzXgARx28DaAmU8VEAFooHBPJ4UnQ4JChBi2kFOkhugdFk0SEI7I5iWsNttdnMCQteDAaFQHk8XkgAEyM-RCVRrADCbHxIGBAFYUg4mgAVZnKQmHcpOACSUG8sFMYFuggAgqrTDBNBt9k0AL4moA)
```sh
--parser yaml
```

**Input:**
```yaml
foo:
  # comment
barr:
  # comment


```

**Output:**
```yaml
foo:
# comment
barr:
  # comment

```

Reply:
Ah, seems attaching logic issue. I'll open an issue in the parser.

Reply:
Issue in `yaml-unist-parser` #275